src/main/asciidoc/dataobjects.adoc

```diff
@@ -60,6 +60,9 @@ set string of allowed auth methods.
  if the server supports them. If null or empty all supported methods may be
```

Comment:
Not sure if this should be included in the PR or not -- but since the build tool modified it and it is under SCC I included it.